.pylintrc

```diff
@@ -270,7 +270,7 @@ ignore-docstrings=yes
 ignore-imports=no
 
 # Minimum lines number of a similarity.
-min-similarity-lines=10
+min-similarity-lines=20
```

Comment:
There's also `.pylintrc37` - and I was wondering why this didn't seem to matter there. I've also forgotten why we have two different ones, and we should perhaps figure that out and whether it's still important because... we don't seem to be using `.pylintrc37`. It's currently referenced in the wrong place, in `build-core-37`:

https://github.com/plotly/dash/blob/74083cea1322aa7682f7e427223fd25b33bcb5ea/.circleci/config.yml#L83-L85

rather than in `lint-unit-37` where it belongs:

https://github.com/plotly/dash/blob/74083cea1322aa7682f7e427223fd25b33bcb5ea/.circleci/config.yml#L31-L32

Not a problem that started here, but we should at least propagate this change to `.pylint37`, and then either use that where it was intended to be used or make an issue to fix this later.

Reply:
I'll make the change in `.pylint37`.

Reply:
fixed in 0c242625 and f8521442